Makefile

```diff
@@ -2,7 +2,7 @@ GOHOSTOS:=$(shell go env GOHOSTOS)
 GOPATH:=$(shell go env GOPATH)
 VERSION=$(shell git describe --tags --always)
 
-DOCKER ?= docker
+DOCKER ?= podman
```

Comment:
Do we have any fallback to docker if podman is not installed? Don't want to hold up the PR, but we have to assume that most people (not us) will have docker.

(We can fix with a future PR if need be.)